script/update_jito_validators.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require File.expand_path('../config/environment', __dir__)
+require 'uri'
+require 'net/http'
+
+
+uri = URI("https://kobe.mainnet.jito.network/api/v1/validators")
+response = Net::HTTP.get_response(uri)
+if response.is_a?(Net::HTTPSuccess)
+  json_response = JSON.parse(response.body)
+  jito_vote_accounts = json_response["validators"].map do |jito_validator|
+    jito_validator["vote_account"] if jito_validator["running_jito"]
+  end
+
+  jito_db_validators = Validator.joins(:vote_accounts)
```

Comment:
We should filter by active vote accounts only. 
Otherwise, we will also pull validators that used to be assocciated with these VoteAccounts in the past, but are using different vote accounts now.
And I would also add a test for this case.